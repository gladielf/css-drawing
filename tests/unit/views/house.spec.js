import { mount } from '@vue/test-utils'
import House from '@/views/house.vue'

describe('House', () => {
  it('should ', () => {
    const wrapper = mount(House)
    expect(wrapper.element).toMatchSnapshot()
  })
})
