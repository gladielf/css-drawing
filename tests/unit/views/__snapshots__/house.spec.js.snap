// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`House should  1`] = `
<div
  class="view"
>
  <div
    class="house"
  />
</div>
`;
